### PR TITLE
Add a UI element to cycle through nick completions on mobile

### PR DESCRIPTION
**Pull request by astorije**
This solves #339.

This was heavily inspired by https://github.com/maxpoulin64/lounge/commit/a877e46.
Clearly not a definitive solution but a good easy start to have and to improve later on.

### Result

nick-cycler

### CODE REVIEW

File: client/css/style.css [hunk 1]
```diff
@@ -1731,6 +1740,11 @@ button {
 	#viewport.menu-open {
 		transition: none;
 	}
+
+	/* Nick cycler should only be displayed on mobile */
```
Comment (maxpoulin64) [inline client/css/style.css]: Actually, I don't really see any harm into having it available on desktop as well. Makes it easy to discover for people that are not used to the tab mechanic.

**Comment by astorije** [comment]
@maxpoulin64, made some changes regarding your comment:

- Rebased with `master`
- Do not hide on mobile anymore
- Consequence of this:
  - Use a `click` event now
  - Added a tooltip
  - Re-focus after clicking the button

Mind confirming/updating your review?

**Comment by williamboman** [comment]
Loses input focus on mobile. Perhaps bypass the mobile detection and just `input.focus()`?

**Comment by astorije** [comment]
> Loses input focus on mobile. Perhaps bypass the mobile detection and just `input.focus()`?

Same issue than #483.
Actual fix for that issue should fix both at the same time.

**Pull request merged**